services/web/server/src/simcore_service_webserver/wallets/_payments_handlers.py

```diff
@@ -342,8 +343,15 @@ async def _pay_with_payment_method(request: web.Request):
         #       we decided not to change the return value to avoid changing the front-end logic
         #       instead we emulate a init-prompt-ack workflow by firing a background task that acks payment
 
+        async def _notify_payment_completed_after_response(app, user_id, payment):
+            # A small delay to send notification just after the response
+            await asyncio.sleep(0.1)
```

Comment:
I see there is a delay but I don't understand why this is required. could you maybe clarify that in the comment?

Reply:
@GitHK you mean that the current comment is not clear?